community/collections/src/main/java/org/neo4j/collection/PrimitiveIntCollections.java

```diff
@@ -32,13 +38,10 @@
 import java.util.function.IntPredicate;
 import java.util.function.LongToIntFunction;
 
-import org.neo4j.collection.primitive.base.Empty;
-
 /**
  * Basic and common primitive int collection utils and manipulations.
  *
  * @see PrimitiveLongCollections
- * @see Primitive
  */
 public class PrimitiveIntCollections
```

Comment:
i think we need to take a chance now go over methods in our collection helpers and clean up those as well since we will not gonna need most of those anymore